spokestack/extensions/webrtc/ans.pyx

```diff
@@ -0,0 +1,59 @@
+"""
+This module contains the AutomaticNoiseSuppression class implemented in Cython
+"""
+# distutils: sources = filter_audio/ns/noise_suppression_x.c
+# distutils: include_dirs = filter_audio/ns/include/
+cimport cans
+cimport numpy as np
+
+
+np.import_array()
+
+cdef class AutomaticNoiseSuppression:
+    cdef cans.NsxHandle *_ans
+    cdef int _sample_rate
+    cdef int _policy
+    cdef int _frame_width
+
+    def __dealloc__(self):
+        cans.WebRtcNsx_Free(self._ans)
+        self._ans = NULL
+
+    def __init__(self, sample_rate=16000, policy=0, **kwargs):
```

Comment:
the policy parameter should be something more friendly to the client
consider creating symbolic constants for these like the vad does